src/sqlfluff/core/rules/doc_decorators.py

```diff
@@ -60,12 +69,16 @@ def document_configuration(cls, ruleset="std"):
         return cls
     # Add final blank line
     config_doc += "\n"
-    # Add the configuration section immediately after the class description
-    # docstring by inserting after the first line break, or first period,
-    # if there is no line break.
-    end_of_class_description = "." if "\n" not in cls.__doc__ else "\n"
 
-    cls.__doc__ = cls.__doc__.replace(end_of_class_description, "\n" + config_doc, 1)
+    if "**Anti-pattern**" in cls.__doc__:
```

Comment:
I think we have another rules saying every rule MUST have an anti-pattern. So not sure we need this `if`/`else` but no harm.

Reply:
Hi @tunetheweb, we have a specific rule in `Rule_016` which you could see in https://github.com/sqlfluff/sqlfluff/blob/88df89f4c5fbfedd250e6aabd1dc69710a499951/src/sqlfluff/rules/L016.py#L26. we would skip the docstring test. and the test as below https://github.com/sqlfluff/sqlfluff/blob/7da5c175e3647359a4ea6d0c192aa23e5e8549d4/test/core/rules/docstring_test.py#L13-L29